pkg/build/builder/daemonless.go

```diff
@@ -536,12 +514,12 @@ func (d *DaemonlessClient) RemoveContainer(opts docker.RemoveContainerOptions) e
 	return err
 }
 
-func (d *DaemonlessClient) PullImage(opts docker.PullImageOptions, auth docker.AuthConfiguration) error {
+func (d *DaemonlessClient) PullImage(opts docker.PullImageOptions, searchPaths []string) error {
```

Comment:
ditto - do we need to do the same for image push?

Reply:
It got discussed @adambkaplan and was deemed unnecessary.

Reply:
for reference:  https://github.com/openshift/builder/pull/102#discussion_r328924879